e2e/cypress/fixtures/markdown/markdown_inline_images_6.md

```diff
@@ -1,3 +1,3 @@
 ### In-line Images
 
-![test image](https://mattermost.org/wp-content/uploads/2016/03/logoHorizontal.png)
+![test image](http://mattermost.com/wp-content/uploads/2022/02/logoHorizontal.png)
```

Comment:
@saturninoabril Changing these URLs will likely break a lot of older tests around message attachments and Markdown images since the old URL used for them won't exist any more. I don't know where else we might need to change these

@cwarnermm Also, a bunch of these are changing https URLs to http which isn't ideal. That seems to be happening most in these E2E tests, but we should probably switch those back to using https

Reply:
@emdecr - The new destination URLs for these are ``http`` and that may be a typo. Can you please confirm? In the event that it's intentional, please note @hmhealey's feedback, and let me know how best to proceed.

Reply:
Confirmed with @emdecr that image assets should be ``https``. I'll ensure they're updated in this PR.